src/assets/css/_main.scss

```diff
@@ -254,10 +254,11 @@ table {
 // ------
 
 code {
-  border-radius: $global-radius;
-  overflow-wrap: break-word;
   background-color: $light-gray;
+  border-radius: $global-radius;
   color: $black;
+  overflow-wrap: break-word;
+  padding: rem-calc(1 5);
```

Comment:
I added `padding` and re-sorted the rules because that's what we do in addons-frontend :p